sql/updates/world/2015_10_01_14938_world.sql

```diff
@@ -0,0 +1,457 @@
+-- Trial of the Champion
+-- by Appled
+-- Thanks to Aokromes for providing sniffs
+
+--
+-- General
+--
+
+-- Disabling mmap for Trial of the Champion instance
+-- creatures are spawned according to sniffs behind an invisible wall and they don't know how to go through it
```

Comment:
Use `AddUnitState(UNIT_STATE_IGNORE_PATHFINDING)` in the unit AI when it's needed, then remove it when it's no longer needed. No need to disable mmaps for the entire dungeon.